lib/src/index.rs

```diff
@@ -1620,7 +1620,11 @@ mod tests {
     use test_case::test_case;
 
     use super::*;
-    use crate::commit_builder::new_change_id;
+    /// TODO: This function is still used in some tests, but shouldn't be. Its
```

Comment:
nit: I'm not sure I agree with the reason for the TODO since this test (and the revset test) doesn't need reproducible change ids, or at least it seems very unlikely that they would fail because of a particular change id. I think something like "TODO: Remove this unnecessary dependency on the uuid crate" better matches my feeling.

Reply:
I rewrote the comment.